aikit/transformers/model_wrapper.py

```diff
@@ -806,6 +806,22 @@ def fit(self, X, y=None, **fit_params):
 
         self._already_fitted = True
         return self
+    
+    
+    @if_delegate_has_method(delegate="_model")
+    @exception_improved_logging
+    def inverse_transform(self, X):
```

Comment:
I don't know if you saw that, but it is a little 'messy' to include the inverse transformation... the wrapper is supposed to handle column, selection, cast and all.... but this the 'inverse_transform' goes into the opposite direction... it won't really work.

In particular self.inverse_transform(self.transform(X)) ... doesn't returns X ... it should return the X part that WAS transformed

I see think it can be usefull... we could 'hide' it also ?